elfi/methods/bo/acquisition.py

```diff
@@ -550,6 +551,188 @@ def _evaluate_logpdf(theta):
         return batch_theta
 
 
+class ExpIntVar(MaxVar):
+    r"""The Expected Integrated Variance (ExpIntVar) acquisition method.
+
+    Essentially, we define a loss function that measures the overall uncertainty
+    in the unnormalised ABC posterior over the parameter space.
+    The value of the loss function depends on the next simulation and thus
+    the next evaluation location \theta^* is chosen to minimise the expected loss.
+
+    \theta_{t+1} = arg min_{\theta^* \in \Theta} L_{1:t}(\theta^*), where
+
+    \Theta is the parameter space, and L is the expected loss function approximated as follows:
+
+    L_{1:t}(\theta^*) \approx 2 * \sum_{i=1}^s (\omega^i * p^2(\theta^i)
+                                * w_{1:t+1})(theta^i, \theta^*), where
+
+    \omega^i is an importance weight,
+    p^2(\theta^i) is the prior squared, and
+    w_{1:t+1})(theta^i, \theta^*) is the expected variance of the unnormalised ABC posterior
+    at \theta^i after running the simulation model with parameter \theta^*
+
+    References
+    ----------
+    [1] arXiv:1704.00520 (Järvenpää et al., 2017)
+
+    """
+
+    def __init__(self, quantile_eps=.01, integration='grid', d_grid=.2,
+                 n_samples_imp=100, iter_imp=2, sampler='nuts', n_samples=2000,
+                 sigma_proposals_metropolis=None, *args, **opts):
+        """Initialise ExpIntVar.
+
+        Parameters
+        ----------
+        quantile_eps : int, optional
+            Quantile of the observed discrepancies used in setting the discrepancy threshold.
+        integration : str, optional
```

Comment:
Now that this is user-selectable, maybe there should be some explanation on the difference, pros and cons of each method.

Reply:
Commented the reason behind the integration choices.

Reply:
This explanation suffices, but, as a side note: IS can be and often is more accurate than grid even in small dimensions but here it is more like that grid can be more reliable because sampling from importance density can fail (e.g. samples come only from one mode even though this density is bimodal). In grid case we always what we ask but we lose accuracy when we place the grid points in zero areas.